roles/foreman_devel/tasks/github_push_ssh.yml

```diff
@@ -9,3 +9,4 @@
     name: 'url."git@github.com:".pushInsteadOf'
     value: 'https://github.com/'
     scope: system
+  when: foreman_devel_github_push_ssh
```

Comment:
I'd prefer to keep this conditional in main. IMHO it's better if the `reviewer` role ensures git as well. It's a standalone role and it should work standalone.